azure-pipelines.yml

```diff
@@ -13,6 +13,7 @@ pr:
   
 variables:
   projects: '**/*.sln'
+  testsProjects: '**/*Tests/*.csproj'
```

Comment:
add this variable for me please: `buildConfiguration: $(buildConfiguration)`

we're gonna test using env variables for the pipeline :)